server_admin/topics/authentication/flows.adoc

```diff
@@ -2,197 +2,228 @@
+* The requirements set within the sub-flows and the executions.
 
-.Adding an authentication execution
-image:{project_images}/Create-authentication-execution.png[]
+Executions have a wide variety of actions, from sending a reset email to validating an OTP. Add executions with the *Add execution* button. If you hover over the
+question mark next to *Provider*, you see a description of the execution.
 
-These can be divided into _automatic executions_ and _interactive executions_. _Automatic executions_ are similar to the `Cookie` execution, and will automatically
-perform their action when they are encountered in the flow. _Interactive executions_ will halt the flow, usually to get some user input. Executions that execute
-successfully will get the _success_ status. This is important, because this is part of whether a flow is successful or not. For example, an empty `Browser` flow
-would not allow anyone to log in. For that it would need at least one execution that successfully evaluates, for example a `Username Password Form` that is correctly
-filled and submitted.
+.Adding an authentication execution
+image:{project_images}/Create-authentication-execution.png[Adding an Authentication Execution]
 
-Sub-flows can be added in top level flow with the `Add flow` button, which opens a `Create Execution Flow` page that is very similar to the `Create Top Level Form`
-page. The only difference is that the `Flow Type` can be either `generic` (like before), or `form`. The `form` type is used to construct a sub-flow that generates
-a single form for the user, like what is done for the built-in `Registration` flow. Sub-flows are a special type of execution that evaluate as successful
-depending on how the executions they contain evaluate (and this includes the evaluation of their contained sub-flows). And the logic of this evaluation
-depends on the Requirement of each execution and sub-flow.
+Two types of executions exist, _automatic executions_ and _interactive executions_. _Automatic executions_ are similar to the *Cookie* execution and will automatically
+perform their action in the flow. _Interactive executions_ halt the flow to get input. Executions executing successfully set their status to _success_.  For a flow to complete, it needs at least one execution with a status of _success_. 
 
-Fully understanding this requires a more complete explanation of how requirements work when evaluating a flow, and this also applies to sub-flows.
-Refer to the <<_execution-requirements, execution requirements section>> above for more details.
+It is possible to add sub-flows to top-level flows with the *Add flow* button. The *Add flow* button opens the *Create Execution Flow* page. This page is similar to the *Create Top Level Form* page, the difference being that the *Flow Type* can be *generic* (default) or *form*. The *form* type constructs a sub-flow that generates a form for the user, similar to the built-in *Registration* flow.
```

Comment:
Generally replace "It is possible to..." with "You can..." 
Change "opens" to "displays"
Change ", the difference being" to ". The difference is .."